collector/go.mod

```diff
@@ -5,6 +5,21 @@ go 1.16
 replace github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents => ./lambdacomponents
 
 require (
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
```

Comment:
Is there any reason not to have these indirect imports? I noticed we were deleting them before.

Reply:
We should keep whatever go adds